src/smart-components/App/stories/index.stories.js

```diff
@@ -389,6 +392,7 @@ export const renderUserProfile = () => fitPageSize(
     renderUserProfile={({ user }) => {
       return user.userId;
     }}
+    dateLocale
```

Comment:
dateLocale doesn't have a default language value, is it what you expected?

Reply:
this is typo